amgut/test/test_add_sample.py

```diff
@@ -133,7 +134,7 @@ def test_post_human(self):
             'email': 'REMOVED',
             'other': None,
             'moldy': None,
-            'participant_name': 'REMOVED-0',
+            'participant_name': 'REMOVED-2',
```

Comment:
This is part of that bug I found where participant names were inconsistent between consent and the barcodes table.